src/sass/pages/static_pages.scss

```diff
@@ -267,6 +267,17 @@ body #not_authenticated_financial ul.checked > li {
             }
         }
     }
+    .timer {
+        .time {
+            .colon {
+                color: $COLOR_BLUE;
```

Comment:
can you try giving them a `primary-color` class instead of adding extra styling?